l10n/en/mozorg/about/shared.ftl

```diff
@@ -15,3 +15,5 @@ about-shared-our-products = Our Products
 about-shared-software-innovations = Software and other innovations designed to advance our mission.
 about-shared-get-involved = Get Involved
 about-shared-volunteer = Become a volunteer contributor in a number of different areas.
+vision-for-the-web = { -brand-name-mozilla }'s vision for the Web
```

Comment:
Not sure, but does the plural here cause any issues for translation? @craigcook do you know?

Reply:
I think we tend to avoid plurals with brand names because some languages do different things with them, but sometimes it can't be avoided. It's probably fine.